server/src/main/java/kr/codesquad/library/controller/BookSearchController.java

```diff
@@ -0,0 +1,39 @@
+package kr.codesquad.library.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import kr.codesquad.library.domain.book.response.BooksByCategoryResponse;
+import kr.codesquad.library.global.api.ApiResult;
+import kr.codesquad.library.service.BookSearchService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static kr.codesquad.library.global.api.ApiResult.OK;
+
+@RequiredArgsConstructor
+@Api
+@RestController
+@RequestMapping("/v1")
+public class BookSearchController {
+
+    private final BookSearchService bookSearchService;
+
+    @ApiOperation(value = "메인페이지 API")
+    @GetMapping("/main")
+    public ApiResult<List<BooksByCategoryResponse>> getMainBooks() {
+
+        return OK(bookSearchService.findMainBooks());
```

Comment:
이건 개인 차이일 것 같은데 전 아래의 방식을 더 선호합니다.

```suggestion
        return ApiResult.OK(bookSearchService.findMainBooks());
```

Reply:
똑같은 ApiResult가 대부분 중복되어서 다른 선임 개발자분들은 중복이 많이 되면 static으로 올리더라구요 저도 이번기회에 올렸습니당